apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -267,6 +216,101 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     }
   }
 
+  private _addParametersForAction<TAction extends HeftActionBase, TParameters>(
+    action: TAction,
+    options: ICustomParameterOptions<TParameters>
+  ): TParameters {
+    const existingParameterNames: Set<string> = new Set(
+      action.parameters.map((parameter) => parameter.longName)
+    );
+    const parametersRecord: Partial<TParameters> = {};
```

Comment:
It's not really a record at this point, especially since you're defining getter properties on it below. Maybe `parametersObject`